### ISSUE TITLE
Can't active 11, 12ch...

### ISSUE THREAD

**Issue by louisFpv** [qba667/FlyPlusI6X#2]
Hi, thx for your great work.
I have updated my FX-i6x by using the updater Integrated.1.0.42.1560. It is success to find 12ch in the Display. However, I cannot active the 11 or 12ch the Menu> System> Aux switches. The max ch in Aux switches menu I can active is 10.
Please help me how to active the 11, 12... aux switches,
Regards

**Comment by mlubryka** [comment]
Maybe I'm wrong but I thought the 11 and 12th aux channels are only for telemetry and they are kind of virtual channels, not for use with switches.

**Comment by louisFpv** [comment]
https://user-images.githubusercontent.com/38834826/39406959-7cdb060c-4bf1-11e8-86dc-1e5b8425f9ed.jpeg

**Comment by mlubryka** [comment]
Yes, I have the same view, but in betaflight I have all aux channels and these two are for rssi/telemetry.

**Comment by qba667** [comment]
@mlubryka you are 100% right I can not include those channels on list because it will break internal arrays handling. 
@louisFpv if all is correctly configured and you are getting SNR or ERROR telemetry data from receiver it should works. Please check on display page what are values of channels 11 and 12.

**Comment by louisFpv** [comment]
@mlubryka  @qba667 Very thanks for your reply. 
I have solved the problem. Since I have not changed the configuration in Betaflight, 
The max aux_channels value need to change.  For the original values of max_aux_channels in BF is 6, the receiver only gets 1500 in Aux7 and 8. 
When all 14 channels enabled, it must change the BF configuration:
set max_aux_channels = 10
It will see the values of Aux7 and 8 changed to Max 2000.  
The difference can be seen in the attached 2 pictures.
Thanks again.

![bfreading](https://user-images.githubusercontent.com/38834826/39943003-e9e694a4-5593-11e8-936d-272de553f03e.JPG)
![bfreading 2](https://user-images.githubusercontent.com/38834826/39943018-f6b06872-5593-11e8-9c11-47ec8fd5d27d.JPG)
